addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -328,7 +328,7 @@ def _timesheet_preprocess(self, vals_list):
                 vals['project_id'] = data.project_id.id
             if not vals.get('account_id'):
                 account_id = data.analytic_account_id.id
-                account = account_per_id[account_id]
+                account = account_per_id.get(account_id)
```

Comment:
Actually, it is a great opportunity to delete my useless code.
```suggestion
                account = data.analytic_account_id
```

Reply:
Nevermind, this doesn't take prject's aa into account